src/Data/List/Base.agda

```diff
@@ -463,6 +465,24 @@ infixl 6 _∷ʳ?_
 _∷ʳ?_ : List A → Maybe A → List A
 xs ∷ʳ? x = maybe′ (xs ∷ʳ_) xs x
 
+------------------------------------------------------------------------
+-- Raw algebraic bundles
+
+module _ (A : Set a) where
```

Comment:
Very minor but can we have an empty line between the module declarations and their contents? I've added it to the list of things to add to the style guide in #1018